doc/plugin_server_bundlepublisher_k8s_configmap.md

```diff
@@ -45,26 +45,31 @@ To use this plugin, configure Kubernetes permissions for the SPIRE Server's Serv
 - For in-cluster SPIRE servers: grant permissions to the Service Account running SPIRE.
 - For out-of-cluster SPIRE servers: grant permissions to the Service Account specified in the kubeconfig.
 
-The Service Account needs permission to:
+The plugin uses the Kubernetes Apply operation to manage ConfigMaps. This operation will create the ConfigMap if it doesn't exist, or update it if it does. The Service Account needs permission to use the `patch` verb on ConfigMaps in the specified namespace.
 
-- `get` and `update` the specific ConfigMap (required).
-- `create` ConfigMaps in the namespace (optional, only if the ConfigMap doesn't exist).
```

Comment:
create is still needed if it doesnt exist for it to work. It does allow restricting to a single resourceName